projboard/tests/test_subject.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from projboard.models.subject import Subject
+
+ANIMALS = "Animals"
+NOTEXIST = "notexist"
+REDHAT = "redhat"
+NAME = "yael"
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def generate_subject():
```

Comment:
Please use two fixtures

If you need 1 use fixture for one if you need more, use another fixture, Please review all the tests

def generate_subject():
    subject = Subject.create_and_get_subject(name="yael")
    subject.save()
    return subject

and another one  to return more
def generate_subjects(generate_subject)
  subject = Subject.create_and_get_subject(name="yael")
    subject.save()
    return [subject, generate_subject]

Reply:
@yaelnoyman1 I assume it is done
Please reply on the conversations

Reply:
I changed the fixture like you asked
and changed the functions